background.ts

```diff
@@ -1000,3 +1004,15 @@ function isRoot(): boolean {
 
   return os.userInfo().uid === 0;
 }
+
+async function runRdctlSetup(newSettings: settings.Settings): Promise<void> {
+  // don't do anything with auto-start configuration if running in development
+  if (isDevEnv) {
+    return;
+  }
+
+  const rdctlPath = path.join(paths.resources, os.platform(), 'bin', 'rdctl');
```

Comment:
Does this need to be `rdctl.exe` on Windows? I think we have a [helper](https://github.com/rancher-sandbox/rancher-desktop/blob/main/pkg/rancher-desktop/utils/resources.ts#L18) for that.

Reply:
Right you are, nice catch